ios/PolyPodApp/PolyPodUITests/OnboardingTest.swift

```diff
@@ -0,0 +1,50 @@
+import XCTest
+
+class OnboardingTest: XCTestCase {
+    let app = XCUIApplication()
+    
+    override func setUp() {
+        continueAfterFailure = false
+    }
+    
+    func testOnboardingShowsUp() {
+        launchApp(firstRun: true)
+        // TODO: Assert onboarding screen is shown
+        XCTAssertTrue(true)
+    }
+
+    private func launchApp(firstRun: Bool = false, resetDefaults: Bool = true) {
```

Comment:
Are you going to use launchApp from different places? Probably so, leaving this just in case.

Reply:
Yeah, similar to how I use it in `UpdateNotificationTest`.